src/foam/nanos/dig/DigFileUploadView.js

```diff
@@ -103,6 +103,11 @@ foam.CLASS({
       name: 'dragActive',
       value: false
     },
+    {
+      class: 'String',
+      name: 'acceptFormat',
+      value: false
```

Comment:
`false` doesn't make sense as a default value for a String property.

Reply:
I removed the value, just wondering should we make this value to 'application/vnd.ms-excel, text/csv'?

Reply:
Yeah, otherwise anything that depended on the default value will break.

Reply:
Ok, changed, please review